### DISCUSSION TITLE
Script Output In Polybar Network Module?

### DISCUSSION THREAD

**Discussion by rhuanpk** [polybar/polybar#3071] [General]
In standard module we has:
```ini
[module/wlan]
type = internal/network
interface-type = wireless
interval = 5
format-connected = <label-connected>
format-disconnected = <label-disconnected>
label-disconnected = disconnected
label-connected = %local_ip%
```

How can I concat a script output to this property (`label-connected`)? e.g.:
```ini
label-connected = %local_ip% $(my-script)
```

**Comment by patrick96** [comment]
This is not possible. Only the script and ipc modules can show arbitrary script output. All other modules are restricted to built-in functionality
